nextflow_schema.json

```diff
@@ -76,6 +76,11 @@
                     "fa_icon": "far fa-file-code",
                     "description": "Path to reference folder"
                 },
+                "genomes_ignore": {
+                    "type": "boolean",
+                    "fa_icon": "far fa-file-code",
+                    "description": "Don't automatically assign reference parameters to the correct references in --genomes_base"
```

Comment:
Is it needed when users use custom directories? if so update update usage

Reply:
No, parameters given by users always have precedence over this. This option should only be used if the automatic assignment of references shouldn't happen